src/ai/storage.py

```diff
@@ -138,20 +136,39 @@ async def store_drone(message: discord.Message, message_copy=None):
         await message.channel.send(f'Drone with ID {target_id} could not be found.')
         return False
 
+    if is_free_storage(drone_to_store):
+        await initiate_drone_storage(drone_to_store, drone_id, target_id, time, purpose, message)
+        return False
```

Comment:
This return is not necessary as the program flow will continue to another return False in line 153 without it as well.